hl-roles/README.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar: auto
+---
+
+# Hacklab Roles 
+![Team](./team.png)
+## Technology Team
+
+### Technology Director
+
+#### Responsibilities
+- Manage the hacklab website repository.
+- Host office hours (with or without appointment) to provide student guidance.
+- Host workshops and develop content for student success.
+- Be responsive on Microsoft Teams to answer inquires about personal projects and career advice.
+- Maintain and oversee the community projects.
+
+#### Qualifications
+- Passionate about technology and sharing that passion through education.
+- A subject matter expert across multiple technical domains.
+- Experienced with conducting technical mentorship.
+- The constant drive to learn and adapt to new technology and environment
+
+## Professional Team
+
+### Career Success Director
+
+##### Qualifications 
+- 1+ years of experience in applying and landing internships/PEY .
+- A deep understanding of  the technical interview process and what makes a strong technical resume.
+- Excellent interpersonal and highly developed communication skills (verbal and written) .
+- Leadership acumen and expereince.
+- Train associates to help support student career growth.
+
+
+#### Responsibilities
+- Hold office hours to help students with resume critiques, career advice, and technical interview questions.
+- Organize and run weekly technical interview workshops: Discuss a problem and facilitate group problem solving.
+- Collaborate with outreach director to curate an online list of career-related resources using Markdown and Vuepress (Ex [Here](https://docs.google.com/document/d/11zgd7rl9bimzh-2R4dDid7wtZui_dNIHCWQdUk23LVY/edit)).
+
+### Outreach Director
+
+#### Qualifications
+- 1+ years of experience in applying and landing internships/PEY .
+- A solid understanding of internship and PEY process.
+- Excellent interpersonal and communication  and skills (verbal and written) .
+- Detail-oriented: Able to handle multiple projects at once and prioritize effectively.
```

Comment:
Not sure if detail oriented is necessary here. Why not just go with The ability to handle ...